src/web/mjs/connectors/VizShonenJump.mjs

```diff
@@ -43,17 +57,29 @@ export default class VizShonenJump extends Connector {
             return;
         }
 
+        data = [...data.querySelectorAll('a.o_chapters-link')];
+
         data = [...data.querySelectorAll('div.o_sort_container div.o_sortable a.o_chapters-link')];
```

Comment:
The method `querySelectorAll` should not be used on an array (`data` is an array at this point, see previous assignment)

Reply:
Yeah I think that was a bad rebase on my part, thanks!